docs/configuration.md

```diff
@@ -180,7 +180,8 @@ information about each option that appears later in this page.
       prefix: /my/nested/registry/
       host: https://myregistryaddress.org:5000
       secret: asecretforlocaldevelopment
-      tls:
+      relativeurls: false
+    tls:
```

Comment:
Is this reindentation of `tls` a mistake? It looks like `tls` should remain underneath `http`.